feg/protos/basic_acct.proto

```diff
@@ -49,16 +49,12 @@ message AcctSession {
         bytes  hardware_addr = 3;
         string name = 4;
     }
-    // ID of the user network (MNO, Identity Provider, etc.)
-    string user_network_id = 5;
     // unique session ID
-    string session_id = 6;
-    // ID of the service provider network
-    string serving_network_id = 7;
+    string session_id = 5;
     // ID of the service provider gateway (AP, AGW, enodeb, etc.)
-    string serving_gateway_id = 8;
+    string serving_apn = 6;
     // available QoS at the caller's site
-    AcctQoS    available_qos = 9;
+    AcctQoS available_qos = 7;
```

Comment:
Regardless of whether this API is guaranteed to be updated uniformly and instantly across our ecosystem - you should avoid changing tag identifiers (here from 9 to 7) as this is not backwards compatible and it's hard to review these changes without system context. Tag identifiers do not have to be in-order within the proto definition. Unless you intend for the new available_qos to be a new field with non-backward-compatibility, leave it at tag 9?

Reply:
It doesn't have to be backward compatible yet.